pkg/modulereader/resreader.go

```diff
@@ -52,6 +52,8 @@ type ModReader interface {
 }
 
 var kinds = map[string]ModReader{
+	// The default kind, when empty, is terraform
+	"":          TFReader{allModInfo: make(map[string]ModuleInfo)},
```

Comment:
My personal preference would be to replace empty string with `terraform` in the expand step so that the expanded blueprint still says `terraform`. WDYT? 

I would approve as is because it is a huge improvement and we could always add back the behavior I describe later, but wondering if you thought about the expanded blueprint.

Reply:
That was, and continues to be my preference as well. I considered 2-3 alternatives and we can talk about them offline. The TLDR is that originally, I thought this could be a one-line change in the reader and so I went for it. I also wanted to add another dependency (defaults). Let me take another look at this.

Reply:
Please, take a look at this alternative now. :)